app/views/home/businesses.html.erb

```diff
@@ -35,14 +35,14 @@
 
   <div class="col-sm-6 col-md-12 float-left float-md-none pl-sm-4 pl-md-0 clearfix">
     <div class="col-12 col-md-6 float-md-right pl-0 pl-md-3 mb-3 mb-sm-0">
-      <h3 class="alt-h3 mb-1">Improve your technology</h3>
+      <h3 class="alt-h3 mb-1">Reduce overhead</h3>
```

Comment:
@nayafia I think custom forks are categorically bad because custom code should be integrated other ways than forking the entire project which means you need to perform a version of the update process yourself for every bug/security release (which usually means: you don't and are insecure/buggy). 

I think my argument for timesaving (not sure how to turn this into good words) is that submitting changes upstream means that they may be accepted which means you don't need to maintain them at all any more but if they are rejected then it points that maintaining something other than a fork is a better use of time (i.e. use their API, use a plugin, etc.) The problem is when people don't even try to upstream changes they are 100% certain to be stuck on that custom fork forever whereas if they try to upstream they have less than a 100% chance of being stuck on that fork.

Does any of that make any sense?